test/e2e/performanceprofile/functests/10_performance_ppc/ppc.go

```diff
@@ -0,0 +1,200 @@
+	"github.com/onsi/gomega/gexec"
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+	"sigs.k8s.io/yaml"
+)
+
+type PPCTest struct {
+	PodmanMakeOptions func(args []string) []string
+	PodmanBinary      string
+}
+
+type PPCSession struct {
+	*gexec.Session
+}
+
+func (p *PPCTest) MakeOptions(args []string) []string {
+	return p.PodmanMakeOptions(args)
+}
+
+func (p *PPCTest) PodmanAsUserBase(args []string, noEvents, noCache bool) (*PPCSession, error) {
+	var command *exec.Cmd
+	podmanOptions := p.MakeOptions(args)
+	podmanBinary := p.PodmanBinary
+	fmt.Printf("Running: %s %s\n", podmanBinary, strings.Join(podmanOptions, " "))
+	command = exec.Command(podmanBinary, podmanOptions...)
+	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+	if err != nil {
+		return nil, err
+	}
+
+	return &PPCSession{session}, nil
+}
+
+type PPCTestIntegration struct {
+	PPCTest
+}
+
+type PPCSessionIntegration struct {
+	*PPCSession
+}
+
+func (p *PPCTestIntegration) MakeOptions(args []string) []string {
+	return args
+}
+func PPCTestCreateUtil() *PPCTestIntegration {
+	p := &PPCTestIntegration{
+		PPCTest: PPCTest{
+			PodmanBinary: DefaultPodmanBinaryPath,
+		},
+	}
+	p.PodmanMakeOptions = p.MakeOptions
+	return p
+}
+
+var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform awareness", func() {
+	mustgatherDir := testutils.MustGatherDir
+	ntoImage := testutils.NtoImageRegistry
+	Context("PPC Sanity Tests", func() {
+		ppcIntgTest := PPCTestCreateUtil()
+		It("[test_id:40940] Performance Profile regression tests", func() {
+			pp := &performancev2.PerformanceProfile{}
+			var reservedCpuCount = 2
+			defaultArgs := []string{
+				"run",
+				"--entrypoint",
+				"performance-profile-creator",
+				"-v",
+			}
+
+			cmdArgs := []string{
+				fmt.Sprintf("%s:%s:z", mustgatherDir, mustgatherDir),
+				ntoImage,
+				"--mcp-name=worker",
+				fmt.Sprintf("--reserved-cpu-count=%d", reservedCpuCount),
+				fmt.Sprintf("--rt-kernel=%t", true),
+				fmt.Sprintf("--power-consumption-mode=%s", "low-latency"),
+				fmt.Sprintf("--split-reserved-cpus-across-numa=%t", false),
+				fmt.Sprintf("--must-gather-dir-path=%s", mustgatherDir),
+			}
+			podmanArgs := []string{}
+			podmanArgs = append(defaultArgs, cmdArgs...)
+			session, err := ppcIntgTest.PodmanAsUserBase(podmanArgs, false, false)
+			Expect(err).ToNot(HaveOccurred(), "Podman command failed")
+			output := session.Wait(20).Out.Contents()
+			err = yaml.Unmarshal(output, pp)
+			Expect(err).ToNot(HaveOccurred(), "Unable to marshal the ppc output")
+			cpuset := *pp.Spec.CPU.Reserved
+			totalReservedCpus := len(strings.Split(string(cpuset), ","))
+			Expect(totalReservedCpus == reservedCpuCount).To(BeTrue())
```

Comment:
you can use gomega To(Equal(..)) instead: 
Expect(ACTUAL).To(Equal(EXPECTED))

Reply:
fixed in the latest commit